TokenAuthIntegTests.testRefreshingInvalidatedToken fails on master

Failed twice today on master:

Log: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+multijob+fast+part2/928/console
Build Scans: https://gradle-enterprise.elastic.co/s/h4jvitsdj4ij2

The failure doesn't reproduce for me:

REPRODUCE WITH: ./gradlew :x-pack:plugin:security:test --tests "org.elasticsearch.xpack.security.authc.TokenAuthIntegTests.testRefreshingInvalidatedToken" \
  -Dtests.seed=93A13A9B7F9E52B9 \
  -Dtests.security.manager=true \
  -Dtests.locale=sw-UG \
  -Dtests.timezone=SystemV/PST8 \
  -Dcompiler.java=12 \
  -Druntime.java=11

```
java.lang.AssertionError: 1 channels still being tracked in HttpChannelTaskHandler while there should be none expected:<0> but was:<1>
```

--- a/test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java
+++ b/test/framework/src/main/java/org/elasticsearch/test/ESIntegTestCase.java
@@ -509,7 +509,7 @@ public abstract class ESIntegTestCase extends ESTestCase {
         return testCluster;
     }
 
-    private static void clearClusters() throws IOException {
+    private static void clearClusters() throws Exception {
         if (!clusters.isEmpty()) {
             IOUtils.close(clusters.values());
             clusters.clear();
@@ -518,9 +518,9 @@ public abstract class ESIntegTestCase extends ESTestCase {
             restClient.close();
             restClient = null;
         }
-        assertEquals(HttpChannelTaskHandler.INSTANCE.getNumChannels() + " channels still being tracked in " +
-            HttpChannelTaskHandler.class.getSimpleName() + " while there should be none", 0,
-            HttpChannelTaskHandler.INSTANCE.getNumChannels());
+        assertBusy(() -> assertEquals(HttpChannelTaskHandler.INSTANCE.getNumChannels() + " channels still being tracked in " +
+                    HttpChannelTaskHandler.class.getSimpleName() + " while there should be none", 0,
+                HttpChannelTaskHandler.INSTANCE.getNumChannels()));
     }
 
     private void afterInternal(boolean afterClass) throws Exception {

